Jenkinsfile

```diff
@@ -0,0 +1,34 @@
+pipeline {
+
+  environment { IMAGE = 'ocpmetal/assisted-installer'}
+  agent {
+    node {
+      label 'bm-inventory-subsystem'
+    }
+  }
+  stages {
+    stage('build') {
+      steps {
+        sh 'docker build . -f Dockerfile.assisted-installer-build -t ocpmetal/assisted-installer'
```

Comment:
Pleas use the Makefile

Reply:
I didn't use  the Make file since it uses skipper
if we use the docker file we are limited to running on the hosts